manager/src/main/java/com/redhat/service/smartevents/manager/ProcessorServiceImpl.java

```diff
@@ -289,16 +314,25 @@ private boolean isProcessorActionable(Processor processor) {
 
     @Override
     public ProcessorDTO toDTO(Processor processor) {
-        String topicName = processor.getType() == ProcessorType.SOURCE
-                ? resourceNamesProvider.getProcessorTopicName(processor.getId())
-                : resourceNamesProvider.getBridgeTopicName(processor.getBridge().getId());
+        String topicName;
```

Comment:
I just extended the existing code to set the `ProcessorDTO` Topic Name.

`ProcessorDTO`'s are used by the Operator to connect a `Executor` to the appropriate topic.